scripts/validate_rd2_enabled.cls

```diff
@@ -0,0 +1,20 @@
+/**
+* @description Validates that Recurring Donations 2 has been enabled in an org. Throws an exception if it hasn't been.
+* Note that this needs to work when a full Managed NPSP package is installed in an org.
+* @author Michael Smith
+* @date 2019-09-17
+*/
+public class SettingsException extends Exception {}
+
+public static Boolean validateRd2Enabled() {
```

Comment:
@davisagli I wasn't quite sure what the task should return to confirm that RD2 is properly enabled. For this draft I just throw an exception if it's not and return true if it is. Is that the best way to handle the validation?

Reply:
Hmm I think I'll need to wrap it in a special Python task that turns exception-or-no-exception into False/True for the Python return value.